code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -771,7 +773,7 @@
 	var/changed_something = FALSE
 	var/obj/item/organ/new_organ = pick(GLOB.bioscrambler_valid_organs)
 	var/obj/item/organ/replaced = get_organ_slot(initial(new_organ.slot))
-	if (!(replaced?.organ_flags & ORGAN_SYNTHETIC))
+	if (!replaced || IS_ROBOTIC_ORGAN(replaced))
```

Comment:
```suggestion
	if (!replaced || !IS_ROBOTIC_ORGAN(replaced))
```
This doesn't seem right?

Reply:
It is right!

Reply:
It wants to replace robotic organs, therefore this is OK.

Reply:
wait shit ur right LMAO

Reply:
wait no, what i did is correct (trust me)
bioscrambler should not affect robotic organs.

Reply:
Intended behavior seems to be to bioscramble an organ slot if it's either empty or doesn't hold a synthethic organ. New code seems to only proc if the slot is empty or has a robotic organ

Reply:
Bro are you sure this doesn;t seem right

Reply:
i'm gonna trust you here Time

Reply:
you can make fun of me if im wrong in the inevitable bug fix PR